packages/blade/src/components/BottomSheet/_decisions/decisions.md

```diff
@@ -0,0 +1,245 @@
+# Bottom Sheet
+
+Bottom sheet is a container that presents additional information to a user in an overlay.
+
+This document outlines the API of `BottomSheet` component.
+
+<img src="./bottom-sheet-thumbnail.png" width="380" alt="Thumbnail of bottom sheet" />
+
+- [Bottom Sheet](#bottom-sheet)
+  - [Design](#design)
+  - [Anatomy](#anatomy)
+  - [API](#api)
+    - [`BottomSheet`](#bottomsheet)
+    - [`BottomSheetBody`](#bottomsheetbody)
+    - [`BottomSheetHeader` API](#bottomsheetheader-api)
+    - [`BottomSheetFooter` API](#bottomsheetfooter-api)
+  - [Composition with Dropdown](#composition-with-dropdown)
+  - [Accessibility](#accessibility)
+  - [Open questions](#open-questions)
+  - [Alternative APIs](#alternative-apis)
+  - [References](#references)
+
+## Design
+
+- [Figma - BottomSheet](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=26477%3A578228&t=1RMqjs99P33Udj0C-0)
+
+## Anatomy
+
+<img src="./bottomsheet-anatomy.png" alt="BottomSheet Anatomy" width="100%" />
+
+
+Components:
+
+- BottomSheet
+  - BottomSheetHeader
+  - BottomSheetBody
+  - BottomSheetFooter
+
+## API
+
+Sample usage:
+
+```jsx
+import { BottomSheet } from '@razorpay/blade';
+
+<BottomSheet isOpen={boolean} snapPoints={[]}>
+  <BottomSheetHeader
+    title="Select Account"
+    leading={<BladeIcon />}
+  />
+  <BottomSheetBody>
+    <Text>Body Content<Text>
+  </BottomSheetBody>
+  <BottomSheetFooter
+    title="Footer Title"
+    leading={BladeIcon}
+    trailing={
+      <>
+        BladeIcon | IconButton | Button
+      </>
+    }
+  />
+</BottomSheet>;
+```
+
+**Design Constraints**
+
+- BottomSheetHeader:
+  - title
+  - leading:
+    - BladeIcon
+
+- BottomSheetFooter:
+  - title
+  - leading:
+    - BladeIcon
+  - trailing
+    - any jsx element
+
+- The BottomSheetFooter will behave similar to ActionListFooter.
+- The Header, Footer can be individually omitted
+
+### `BottomSheet`
+
+We'll expose a `BottomSheet` component with the following API:
+
+| Prop            | Type        | Default                 | Description                                                                                     | Required |
+| --------------- | ----------- | ----------------------- | ----------------------------------------------------------------------------------------------- | -------- |
+| isOpen            | `boolean`   | `false`                 | Toggles bottom sheet state                                                              |          |
+| snapPoints      | `[number, number, number]`  | `[0.35, 0.5, 0.85]` | Snappoints in which the bottom sheeet will rest on, this accepts a number between 0 & 1 which maps to the total view height of the screen. 0.5 means 50% of screen height.                                              |          |
```

Comment:
when snappoints is not provided what is the behavior? what will be the height of the bottomsheet? also do i need to pass all the snap points? what if I only pass one snappoint? will the initial height of the bottomsheet will be equal to this number?